core/templates/dev/head/pages/question_editor/QuestionEditorDirective.js

```diff
@@ -99,9 +100,11 @@ oppia.directive('questionEditor', [
           };
 
           var _updateQuestion = function(updateFunction) {
+            if ($scope.questionChanged) {
```

Comment:
Just checking, is this just checking that the function is set? What happens if this check isn't there?

Reply:
There are some question editor calls present currently (I think in creator dashboard, where in the future anyone can suggest a question) which doesn't use this now, so that check is there to not throw an error for those calls.